forte/processors/base/batch_processor.py

```diff
@@ -179,17 +174,27 @@ def predict(self, data_batch: Dict) -> Dict:
         """
         pass
 
-    def pack_all(self, output_dict: Dict):
+    def pack_all(self, packs: List[DataPack], output_dict: Dict):
```

Comment:
I believe there are other classes that are using this function? Maybe need to add something there too? Otherwise the travis may break.

Reply:
I passed the test on pipeline. I will search if there are any other place.

Reply:
It seems this function is only used in this class. We will see whether travis works.

Reply:
If you use some IDE, you should be able to find the usage of it. Otherwise, you can search for the function. Or travis will tell you if it breaks.

Reply:
Yes. And this function should only used in this file.